source/jobs/senior-product-designer.html.erb

```diff
@@ -8,7 +8,7 @@ header:
   image: "/jobs/images/twitter-card-image-design.jpg"
 
 introduction:
-  title: "This vacancy is no longer open for applications"
+  title: "Are you a Senior Digital Product Designer?"
```

Comment:
I think this is the wrong file? Isn't there a senior ux designer file in there somewhere?

Reply:
> I think this is the wrong file? Isn't there a senior ux designer file in there somewhere?

Yes, this is not the right file ... I made a comment on Slack about it yesterday:

> sorry, I think I’ve linked to the wrong job page. It should be this one: https://deploy-preview-643--unboxed-co-production.netlify.app/jobs/senior-product-ux-designer

Reply:
I'm a bit confused. There are two job ads - senior product designer, and senior ux designer. It now links to the correct one, but why has this page (senior product designer) been edited if it's not the one we're using.

Reply:
It might be because I posted a link to it on Slack initially, but that was a mistake, sorry.

Reply:
You can delete the 'senior product designer' one. And just keep the 'senior product (ux) designer'.

Reply:
Removed all changes from that file